<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="HomeBut">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>740</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>951</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="OptionBut1">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Option 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OptionBut2">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Option 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OptionBut3">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Option 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OptionBut4">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Option 4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="IDKBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I don't know </string>
   </property>
  </widget>
  <widget class="QLabel" name="ResultLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>600</y>
     <width>881</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="QuestionBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>841</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The question will be displayed here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextQuestionBut">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>700</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Next Question</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
